virtual_ecosystem/core/core_components.py

```diff
@@ -85,6 +85,8 @@ class ModelTiming:
     """The configured run length as a pint Quantity."""
     update_interval: np.timedelta64 = field(init=False)
     """The configured update interval."""
+    update_interval_seconds: float = field(init=False)
+    """The configured update interval in seconds."""
```

Comment:
I like this very much!

Reply:
Seemed like a useful thing that all models might need! Might advertise it once this merges!